exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
+  ["",                         ""],
+  // single C nucleotide strand
+  ["C",                       "G"],
+  // single G nucleotide strand
+  ["G",                       "C"],
+  // single A nucleotide strand
+  ["A",                       "U"],
+  // single T nucleotide strand
+  ["T",                       "A"],
+  // longer strand
+  ["ACGTGGTCTTAA", "UGCACCAGAAUU"],
+  // strand with invalid nucleotides
+  ["XCGFGGTDTTAA",        Error()],
+  // strand with invalid nucleotides after the first nucleotide
+  ["ACGTFGTBTEAA",        Error()]
+};
+
+test
+parameters(`value cases`)
+void testHamming(String dna, String|Error expected) {
+  if(is String expected) {
```

Comment:
Haha; there's a style checker at work that insists of having spaces between the if and parenthesis and it makes me _so_ angry, every time. But I have no qualms with changing it. It's community code and understandable to have it be consistent. I'll get it altered.